examples/microsoft/teams/index.js

```diff
@@ -0,0 +1,21 @@
+const { Reshuffle } = require('reshuffle')
+const { TeamsConnector } = require('reshuffle-microsoft-connectors')
+
+const app = new Reshuffle()
+const connector = new TeamsConnector(app, {
+  AppId: process.env.AppId,
+  AppPassword: process.env.AppPassword,
+  AppTenantId: process.env.AppTenantId,
+})
+
+connector.on(
+  {
+    resource: '/teams/getAllMessages',
+    changeType: 'created',
+    runtimeBaseUrl: 'https://example.com',
+    expirationDateTime: '2021-02-03T03:47:17.292Z',
```

Comment:
how about setting `expirationDateTime` to 60 minutes ago so you don't get too many messages.
e.g. `const sixtyMinutesAgo = (new Date(Date.now() - 1000 * 60 * 60)).toISOString()`